webapp/channels/src/sass/components/_post-right.scss

```diff
@@ -163,7 +163,7 @@
             margin-top: 4px;
```

Comment:
Looks like this might be causing this to sit too close to the bottom edge of the screen when viewing a long thread that pushes the input box to the bottom.

suggest changing the CSS to match the center channel input:
```
margin: 2px 0;
line-height: 16px;
```

<img width="431" alt="image" src="https://github.com/mattermost/mattermost/assets/2040554/b6457374-6c08-4f5f-9f98-8275e1c8a646">

Reply:
@matthewbirtch I've applied those CSS changes. Take a look now and let me know if it's good